gulp/makeCss.js

```diff
@@ -62,3 +62,7 @@ gulp.task("copyCssDeps", function() {
     )
     .pipe(gulp.dest("./src/scss/cssIncludes"));
 });
+gulp.task('copyCssImages', function() {
+   gulp.src('./node_modules/leaflet/dist/images/*.png')
+   .pipe(gulp.dest(paths.bundleDir+'/images'));
+});
```

Comment:
not sure about the css images. Is there a need to copy them? Re-using the YASR lib as dep would require you to serve these images as well from that specific path